recipes/crbeam/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "crbeam" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/okolo/mcray/archive/refs/tags/v1.0.6rc1.tar.gz
+    sha256: 0ef92a6116394dd7926f0a7ee446a6d9ef7c79c904b63d6689fe49751ffae035
+build:
+  skip: True  # [win or osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake ==3.22.3
```

This pin looks too strict.
```suggestion
    - cmake
```